client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -56,7 +105,13 @@ interface ConnectedOrganization {
   handle: ConnectionHandle;
 }
 
+const userInfo: Ref<ClientInfo | null> = ref(null);
+const currentDevice = ref<AvailableDevice | null>(null);
+const isTrialOrg = ref(false);
 const connectedOrgs: Ref<Array<ConnectedOrganization>> = ref([]);
+const currentOrg: Ref<ConnectedOrganization | undefined> = ref(undefined);
+const filteredOrgs: Ref<Array<ConnectedOrganization>> = ref([]);
+const expirationDuration = ref<Duration | undefined>(undefined);
```

Comment:
This is currently unused, it might be useful if we implement a function as suggested in the previous comment, otherwise the duration of the current organization is already displayed on the sidebar